tests/Backend/SOX/MergeRequestsTest.php

```diff
@@ -23,25 +23,33 @@ public function setUp(): void
         $this->prodManagerClient = $this->getDefaultClient();
         $this->developerClient = $this->getDeveloperStorageApiClient();
         $this->branches = new DevBranches($this->developerClient);
-        foreach ($this->branches->listBranches() as $branch) {
-            if ($branch['isDefault'] !== true) {
-                $this->branches->deleteBranch($branch['id']);
-            }
+        foreach ($this->getBranchesForCurrentTestCase() as $branch) {
+            $this->branches->deleteBranch($branch['id']);
         }
 
         $this->cleanupConfigurations($this->getDefaultBranchStorageApiClient());
     }
 
-    public function testCreateMergeRequest(): void
+    private function getBranchesForCurrentTestCase(): array
     {
-        $oldBranches = $this->branches->listBranches();
-        $this->assertCount(1, $oldBranches);
```

Comment:
Toto bylo hrozně zavádějící všude, prostě teď vytahuju všude default branch a a mažu branche enom pro konkrétní test case, nemá smysl to assertovat.